src/ploigos_step_runner/step_implementers/uat/maven_integration_test.py

```diff
@@ -179,6 +171,53 @@ def _run_step(self):
         # return result
         return step_result
 
+    def __get_target_host_url(self):
```

Comment:
@cthiel1 rather then have two return results, why not just pass in the step_result as a param?